File: Tower_of_Hanoi.py
def Tower_Hanoi(disks,source,dest,aux):
    if(disks==1):
        print('Disk no.',disks,'moved from',source,'to',dest)
        return
    Tower_Hanoi(disks-1,source,aux,dest)
    print('Disk no.',disks,'moved from',source,'to',dest)
    Tower_Hanoi(disks-1,aux,dest,source)
Tower_Hanoi(3,'A','B','C')

File: Tower_of_Hanoi.js
function Tower_Hanoi(disks,source_rod,des_rod,aux_rod){
    if(disks===1){
        console.log("Move disk 1 from rod",source_rod,"to rod",des_rod);
        return;
    }
    Tower_Hanoi(disks-1,source_rod,aux_rod,des_rod);
    console.log("Move disk",disks,"from rod",source_rod,"to rod",des_rod);
    Tower_Hanoi(disks-1,aux_rod,des_rod,source_rod);
    
}
let disks=2;
Tower_Hanoi(disks,"A","B","C");
